packages/teams-js/test/public/meetingLegacy.spec.ts

```diff
@@ -135,16 +135,16 @@ describe('meeting_V1', () => {
 
```

Comment:
As discussed offline, it looks like meeting.spec.ts is a superset of this file after the changes to meeting.ts. I think this file can be deleted.